### PR TITLE
Update for  UI improvements and handling API fetching error

### PR DESCRIPTION
**Pull request by duy-nguyentuong**
- Update API fetching codes to handle error
- Update `get` method ( for fetching data from API) to have better error handling
- Fix/Update UI issues in QA reviews list: https://www.notion.so/FE-QA-reviews-5f4d43a65dce4728a9c22de182d02855

### CODE REVIEW

**Comment by Git-on-my-level** [comment]
LGTM but I'm probably not the best person to review this

File: ts/prime/ApiReact/src/elements/label.tsx [hunk 1]
```diff
@@ -21,8 +22,13 @@ export interface InputFeedbackProps {
 }
 
 export const InputFeedback: React.FC<InputFeedbackProps> = (props: InputFeedbackProps) => {
+  const { Text } = Typography
   const { error } = props
-  return error ? <div className="input-feedback">{error}</div> : null
+  return error ? (
+    <Text className="input-feedback" type="danger">
```
Comment (chauhannishith) [inline ts/prime/ApiReact/src/elements/label.tsx:28]: why this change? why are we using [this component](https://ant.design/components/typography/#components-typography-demo-text) instead of just the css?

Comment (duy-nguyentuong) [reply ts/prime/ApiReact/src/elements/label.tsx:28]: @chauhannishith I felt that we're creating a lot of stuff that was existing in `antd` lib. By this way, we can reduce a lot of CSS codes added and unify the UI for the product.

Comment (chauhannishith) [reply ts/prime/ApiReact/src/elements/label.tsx:28]: yes but this is in the api

File: ts/prime/Web/src/blocks/Form/index.js [hunk 2]
```diff
@@ -19,8 +37,13 @@ const Label = (props) => {
 }
 
 const InputFeedback = (props) => {
+  const { Text } = Typography
   const { error } = props
-  return error ? <div className="input-feedback">{error}</div> : null
+  return error ? (
+    <Text className="input-feedback" type="danger">
```
Comment (chauhannishith) [inline ts/prime/Web/src/blocks/Form/index.js:43]: you have already made the same change here

File: ts/prime/ApiReact/src/elements/label.tsx (hunk 1 above)
Comment (duy-nguyentuong) [reply ts/prime/ApiReact/src/elements/label.tsx:28]: Yes, but I guess there's somewhere use these components. I've searched with the `className="input-feedback"` and update them.
Is there any side effects of this change?

Comment (chauhannishith) [reply ts/prime/ApiReact/src/elements/label.tsx:28]: can not term as a side effect, you can make the change but i don't know if we want to add antd to our package

Comment (duy-nguyentuong) [reply ts/prime/ApiReact/src/elements/label.tsx]: Removed changes in ApiReact and update other place to use existing InputFeedback in `prime/Web`

File: ts/prime/ApiReact/src/hooks/effects/fetch.ts [hunk 3]
```diff
@@ -16,12 +16,17 @@ export function useFetch<T extends FetchType<T['message']>>(
   useEffect(() => {
     const initialValues = async (): Promise<void> => {
       if (!action.getUrl) return
-      try {
-        const getValues = await auth.api().get(action.getUrl)
-        dispatchValues(new action.messageType(getValues))
-      } catch (e) {
-        setError(e.message)
+
+      const response = await auth.api().get(action.getUrl)
```
Comment (chauhannishith) [inline ts/prime/ApiReact/src/hooks/effects/fetch.ts:20]: i think keeping this in try catch is still better, just as a fail safe

**Comment by chauhannishith** [comment]
LGTM

File: ts/prime/ApiReact/src/hooks/effects/fetch.ts (hunk 3 above)
Comment (duy-nguyentuong) [reply ts/prime/ApiReact/src/hooks/effects/fetch.ts:20]: Since we already threw the error response in `get()` method ([see here](https://github.com/stablyio/ts-packages/blob/738b601d510a1702034718776dd31d98af3d9f14/ts/prime/Api/src/services/index.ts#L75)), now it should be fine here.

File: ts/prime/Web/src/components/transfer/withdrawalUSDS.tsx [hunk 4]
```diff
@@ -30,50 +35,54 @@ const WithdrawalUSDSComponent = ({
...
-                  />
-                </Row>
-                <Row className="input-row">
-                  <Button type="primary" size="large" className="custom-btn-primary" onClick={handleSubmit}>
-                    Transfer
-                  </Button>
-                </Row>
-              </form>
-            </Card>
+            <Spin tip="Loading..." spinning={false}>
```
Comment (dhanesh-kapadiya) [inline ts/prime/Web/src/components/transfer/withdrawalUSDS.tsx:38]: pass loading variable ?

Comment (duy-nguyentuong) [reply ts/prime/Web/src/components/transfer/withdrawalUSDS.tsx:38]: I will update this when implement USDS withdrawal API integration.
